backend/data/src/main/kotlin/io/tolgee/formats/xliff/in/Xliff12FileProcessor.kt

```diff
@@ -50,7 +51,13 @@ class Xliff12FileProcessor(
     transUnitId: String,
     language: String,
   ) {
-    val converted = convertMessage(text, language)
+    val textToConvert =
```

Comment:
I don't think it makes any sense to escape html when importing. This wasn't in the scope of the task.

Reply:
Will remove it, I only added it since the author of the issue mentioned it

<img width="670" alt="image" src="https://github.com/user-attachments/assets/8e10b4fa-5563-4393-97a0-245c60867fa3" />